File: 백준/Bronze/14626. ISBN/ISBN.cc
#include<bits/stdc++.h>
using namespace std;
using ll = long long;

int main(){
    cin.tie(0); cout.tie(0); ios::sync_with_stdio(false);
    string isbn; cin >> isbn;
    int target, total = 0;
    for(int i = 0; i < 13; i++){
        if(isbn[i] == '*'){ 
            target = i; continue;
        }
        total += (isbn[i]-'0')*(i % 2 ? 3 : 1);
    }
    for(int i = 0; i < 10; i++){
        if((total + i*(target % 2 ? 3 : 1)) % 10 == 0){
            cout << i; break;
        }
    }
    return 0;
}

File: 백준/Bronze/14626. ISBN/ISBN.py
import sys
def print(*args, sep=" ", end="\n"): return sys.stdout.write(sep.join(map(str, args)) + end)
def input(): return sys.stdin.readline().rstrip()

def main():
    isbn = input()
    total = 0
    for i in range(13):
        if isbn[i] == '*': target = i; continue
        total += int(isbn[i])*(3 if i % 2 else 1)
    for i in range(10):
        if (total + i*(3 if target % 2 else 1)) % 10 == 0:
            print(i); break
    return 0
if __name__ == '__main__':
    sys.exit(main())